CODEOWNERS

```diff
@@ -30,8 +30,7 @@
 Kconfig*                                  @tejlmand
 # All doc related files
 /doc/CMakeLists.txt                       @carlescufi
-/doc/scripts/                             @carlescufi
-/doc/static/                              @carlescufi
+/doc/scripts/                             @carlescufi /doc/static/                              @carlescufi
```

Comment:
```suggestion
/doc/scripts/                             @carlescufi
/doc/static/                               @carlescufi
```

Reply:
not sure about the formatting on this suggested change, so double check that.

Reply:
Woops that was nnot how it supposed to look guess I modified it after :/